### PR TITLE
Hit api

### PR DESCRIPTION
**Pull request by phelian23**
In this pull request
- API documentation was used
- I send and receive data from API

### CODE REVIEW

File: src/scores.js [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
+  (await fetch(baseUrl, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify(newScore),
+  })).json();
+};
+
+function displaySores(scores) {
```
Comment (Mihndim2020) [inline src/scores.js]: - [ ] As it is a requirement for this project and best practice, please kindly use `arrow functions` not function key word. 
phe

**Review by Mihndim2020 — APPROVED** [review]
### Hi @phelian23, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean code
:heavy_check_mark: The project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scores.js (hunk 1 above)
Comment (phelian23) [reply src/scores.js]: Thanks for the review, I'll implement that

**Review by ZeenatLawal — APPROVED** [review]
Hi @phelian23, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**